frontend/apps/app/components/ProjectNewPage/components/WatchSchemaForm/WatchSchemaForm.tsx

```diff
@@ -0,0 +1,76 @@
+'use client'
+import { RemoveButton } from '@liam-hq/ui'
+import type { FormatType } from 'components/FormatIcon'
+import { FormatSelectDropdown } from 'features/sessions/components/shared/FormatSelectDropdown'
+import { ChevronRight } from 'lucide-react'
+import Link from 'next/link'
+import { type ChangeEvent, type FC, useCallback } from 'react'
+import styles from './WatchSchemaForm.module.css'
+
+type Props = {
+  filePath: string
+  format: FormatType
+  disabled?: boolean
+  onFilePathChange: (path: string) => void
+  onFormatChange: (format: FormatType) => void
+}
+
+export const WatchSchemaForm: FC<Props> = ({
+  filePath,
+  format,
+  disabled,
+  onFilePathChange,
+  onFormatChange,
+}) => {
+  const handleChangeFilePath = useCallback(
+    (e: ChangeEvent<HTMLInputElement>) => {
+      onFilePathChange(e.target.value)
+    },
+    [onFilePathChange],
+  )
+
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.item}>
+        <div className={styles.itemHeader}>
+          <div className={styles.itemTitle}>Select Schema Format</div>
+          <FormatSelectDropdown
+            selectedFormat={format}
+            onFormatChange={onFormatChange}
+          />
+        </div>
+      </div>
+      <div className={styles.item}>
+        <div className={styles.itemHeader}>
+          <div className={styles.itemTitle}>Set Watch Schema File Path</div>
```

Comment:
The concept of `Watch` is a remnant from the Liam Migration era and is confusing to users, so I want to remove it. What do you think?